models.py

```diff
@@ -296,6 +299,12 @@ def exists(cls, session, user, txid):
     @classmethod
     def update_wallet_address(cls, session, user):
         if user.wallet_address:
+            # select the merchant_rate to use
+            if user.merchant_rate:
```

Comment:
you can use: `rate = user.merchant_rate if user.merchant_rate else app.config["MERCHANT_RATE"]`

Reply:
changed.